qubes/device_protocol.py

```diff
@@ -733,6 +733,14 @@ def unknown(cls) -> "DeviceInterface":
         """Value for unknown device interface."""
         return cls("?******")
 
+    @staticmethod
+    def from_str_bulk(interfaces: Optional[str]) -> List["DeviceInterface"]:
+        interfaces = interfaces or []
+        return [
+            DeviceInterface(interfaces[i: i + 7])
+            for i in range(0, len(interfaces), 7)
+        ]
```

Comment:
I suggest raising a `QubesValueError` if the size is not a multiple of 8 or if there are invalid characters in the interface name.